src/Components/Web.JS/src/DomWrapper.ts

```diff
@@ -0,0 +1,9 @@
+import '@microsoft/dotnet-js-interop';
+
+export const domFunctions = {
+  focus,
+};
+
+function focus(element: HTMLElement): void {
+  element.focus();
```

Comment:
@SteveSandersonMS don't we have to protect here against bad elementreferences?

Reply:
👍  Good point! It would be good to check that `element instanceof Element` or similar, throwing a clear error if not.

It will be `null` (I think) if the element it referred to has been removed from the document.